list.ts

```diff
@@ -1,8 +1,13 @@
-import { Scenario, seenScenarios } from '.';
-import { sync as globSync } from 'glob';
+import { Scenario, seenScenarios } from './index.js';
+import glob from 'glob';
 import { resolve } from 'path';
 import { format } from 'util';
 
+import { createRequire } from 'node:module';
```

Comment:
I want to find a lint for this, but I like having all the node builtins prefixed with `node:` -- good for differentiating between builtin and installed deps